docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
> Inconsistent coding style seriously harms readability and maintainability since all new contributors 

I agree, for this PR `null` is more appropriate. Still, I'm wondering for the future.